src/app/personside/dialogpanel/sendMelding/autofullforUtils.ts

```diff
@@ -116,16 +117,16 @@ export function autofullfor(tekst: string, autofullforMap: AutofullforMap): stri
 }
 
 export function useAutoFullforData(): AutofullforData | undefined {
-    const personResponse = useHentPersondata();
+    const personResponse = persondataResource.useFetch();
     const saksbehandler = innloggetSaksbehandler.useFetch();
     const enheterResource = saksbehandlersEnheter.useFetch();
-    const enheter = hasData(enheterResource) ? enheterResource.data.enhetliste : [];
+    const enheter = enheterResource.data ? enheterResource.data.enhetliste : [];
     const valgtEnhetId = useValgtenhet().enhetId;
     const valgtEnhet = enheter.find((enhet) => enhet.enhetId === valgtEnhetId);
 
     return {
         enhet: valgtEnhet,
-        person: hasData(personResponse) ? personResponse.data : undefined,
-        saksbehandler: hasData(saksbehandler) ? saksbehandler.data : undefined
+        person: personResponse.data ? personResponse.data : undefined,
+        saksbehandler: saksbehandler.data ? saksbehandler.data : undefined
```

Comment:
kan personResponse.data bli noe annet enn undefined?
```suggestion
        person: personResponse.data,
        saksbehandler: saksbehandler.data,
```